resources/ebsco_kb.php

```diff
@@ -0,0 +1,232 @@
+                                          echo "<option value='$key' $selected>".$type['selectDisplay']."</option>";
+                                        }
+                                    ?>
+                                </select>
+                                <div id="limitBy" style="margin-top: 1em; font-size: .75em;">
+                                    <label><strong><?php echo _($limitLabel);?></strong></label>
+                                    <br />
+                                    <button type="button" id="removeLimit" class="btn">
+                                        <i class="fa fa-times"></i>
+                                    </button>
+                                    <span id="limitName"><?php echo $limitName; ?></span>
+                                </div>
+                            </td>
+                        </tr>
+
+                        <tr class="ebsco-toggle-option titles-option packages-option vendors-option">
+                            <td class='searchRow'>
+                                <div class="ebsco-toggle-option titles-option">
+                                    <select
+                                        name='search[searchfield]'
+                                        id='searchfield'
+                                        style='width:150px'
+                                        data-default="<?php echo EbscoKbService::$defaultSearchParameters['searchfield']; ?>">
+                                        <?php
+                                        foreach(EbscoKbService::$titleSearchFieldOptions as $key => $type){
+                                            $selected = $search['searchfield'] == $key ? 'selected' : '';
+                                            echo "<option value='$key' $selected>$type</option>";
+                                        }
+                                        ?>
+                                    </select>
+                                </div>
+                                <div>
+                                    <label
+                                        for='searchName'
+                                        class="ebsco-toggle-option titles-option">
+                                        <strong><?php echo _("contains");?></strong>
+                                    </label>
+                                    <label
+                                            for='searchName'
+                                            class="ebsco-toggle-option packages-option vendors-option">
+                                        <strong><?php echo _("Name (contains)");?></strong>
+                                    </label>
+                                    <br />
+                                    <?php echo Html::text_search_field_tag('search', isset($search['name']) ? $search['name'] : '' ); ?>
```

Comment:
I got the message
> PHP Deprecated:  Non-static method Html::text_search_field_tag() should not be called statically

when loading the ebsco_kb.php page.

Reply:
fixed. But FYI - I was following the convention on the [index page](https://github.com/coral-erm/coral/blob/development/resources/index.php#L77), which uses this a lot.